Tasks/Karachev/Task5/Task5.py

```diff
@@ -0,0 +1,42 @@
+
+def check_str(l):          
+    k = 0
+    t = 0
+    for i in l:               
+        if i.islower():
+            k+=1
+        elif i.isupper():
+            t+=1        
+    return print(f"{t} upper case, {k} lower case")
+
+check_str("The quick BrBBBow Fox") 
+
+       
+
+def is_prime(x):
+    d = 2
+    while x%d !=0:
+      d+=1
+    return  d == x
```

Comment:
very quick solution in terms of implementing, but not the fastest in terms of work